proto/cosmos/group/v1beta1/query.proto

```diff
@@ -0,0 +1,244 @@
+syntax = "proto3";
+
+package cosmos.group.v1beta1;
```

Comment:
Let's not release any more beta protos to production. We can keep this as is for now _as long as_ we make sure to change to v1 after beta QA.

Reply:
sounds good